_data/api-swagger.yaml

```diff
@@ -1053,6 +1054,28 @@ definitions:
       For example, in the Netherlands, personal number (BSN) can not be used for privacy reasons
       and is substituted by document number such as passport number, but in other countries this poses no problem.
 
+  OpenID Providers:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      description: |
+        Adds an OpenID verification step to the signing process.
```

Comment:
rename to OpenId Connect

Reply:
In our api we call it OpenID Providers. Our friendly name is openID, so good to have one term for both api, portal and documentation.